### ISSUE TITLE
Problem using in termux. 

### ISSUE THREAD

**Issue by shin-illua** [radare/radare2#13179]
I installed radare2 on termux android using `pkg install radare2`, everything was fine, I saw no errors.

But when I run a command like `r2 putty.exe` or `rabin2 putty.exe` or even `r2 -h` it shows nothing.

https://imgur.com/tOXszay

Please help.

**Comment by solomonbstoner** [comment]
I had this problem before too on my phone. I believe the problem lies in `pkg`. I solved it by installing radare2 from source code.

1. `git clone https://github.com/radare/radare2`
2. `sys/user.sh` (this installs for the user instead of for the system as the latter requires root access, which I doubt you would have on your phone).

radare2 should work fine afterwards.

**Comment by radare** [comment]
Can you tell the version of android and cpu of this device? Did you checked the syslog?

**Comment by radare** [comment]
Also termux didnt updated to 3.3 yet. So this version is old. And in any case. This must be an issue in termux. Not in r2 and it works fine in all my androids. So cant repro

**Comment by shin-illua** [comment]
solomonbstoner I executed same commands as you stated above but I get an error `bash: sys/user.sh: Permission Denied` Anyway around it?

**Comment by radare** [comment]
Thats because the hashbanh doesnt matches android paths, just pass it as argument to bash

**Comment by trufae** [comment]
just update, r2-3.3 is in termux now

**Comment by shin-illua** [comment]
@trufae just did it and still nothing.

@radare I dunno, but doing the `ls` command inside `radare2/libr` folder doesn't give me any config.mk file, only `config.mk.tail, config.mk.head`

**Comment by radare** [comment]
in the screenshot you are not doing r2 putty.exe ... i only have 64bit android devices to test and it works fine in all of them, so maybe the issue is related to 32bit arm and termux, did you treid other tools instead of just rabin2?

**Comment by shin-illua** [comment]
@radare yes I did the `apt-get install radare2` command again, and tried putty.exe with r2 and rabin and nothing happened. 

About the missing `config.mk` file, is there anything else I could do about it?

**Comment by radare** [comment]
you may report this issue to termux, i see no reason for not being working.

About the missing config.mk, this is generated by the configure, you can also use the meson build system insitead of configure +make, but both should work

**Comment by shin-illua** [comment]
@radare "generated by configure", does that mean that `config.mk` is supposed to be generated after I run the `sys/user.sh` script? Or do I have to run it manually myself?

**Comment by shin-illua** [comment]
@radare I guess this info will be useful. This is one of the reasons why I think this is not because of termux. 

I installed this app called "radare2 material" from google play. So I clicked the `install` button and it started downloading something and did some installations. Now when I am on this window (the window where  I get to launch radare2, which also has options for me if in what way I want to launch radare2, options being "Console", "Terminal", "Termux", "Webview", "Browser" ) and I try to choose the "Terminal" then it spawns an instance of terminal emulator (https://play.google.com/store/apps/details?id=jackpal.androidterm&hl=en&referrer=utm_source%3Dgoogle%26utm_medium%3Dorganic%26utm_term%3Dterminal+emulator+android&pcampaignid=APPU_1_viGBXNXPNsWqoAS695bQBw) with the `r2 /system/bin/ls` command already entered but still, nothing happens.
